### PR TITLE
feat: remove ULP

### PR DESCRIPTION
**Pull request by cajubelt**
adding a hidden /add-liquidity-legacy page that lets users remove their ULP if they provided liquidity before we switched to using the broker bot

### CODE REVIEW

File: src/pages/ProvideLiquidity/index.tsx [hunk 1]
```diff
@@ -68,30 +86,42 @@ export default function ProvideLiquidity() {
     return <ChangeNetworkModal />
   }
 
+  const cardParams: { title: string; body: string; cta: string; url: string } = showLegacyAddLiquidity
+    ? {
+        title: t('legacyLiquidityProviderCardTitle'),
+        body: t('legacyLiquidityProviderCardBody'),
+        cta: t('legacyLiquidityProviderCardCTA'),
+        url: 'https://revo.market/#/add-liquidity-legacy',
```
Comment (jophish) [inline src/pages/ProvideLiquidity/index.tsx]: Whenever I click this, it redirects to the zap page for some reason

File: src/pages/ProvideLiquidity/index.tsx [hunk 2]
```diff
@@ -68,30 +86,42 @@ export default function ProvideLiquidity() {
     return <ChangeNetworkModal />
   }
 
+  const cardParams: { title: string; body: string; cta: string; url: string } = showLegacyAddLiquidity
```
Comment (jophish) [inline src/pages/ProvideLiquidity/index.tsx:89]: nit: Should we show *both* cards if we detect that the user already has liquidity, and maybe have this one in a  different color so it pops out a bit more? (not totally sure if showing both is worth, but might still be helpful to make this warning stand out a bit)

File: public/locales/en.json [hunk 3]
```diff
@@ -98,6 +98,9 @@
   "liquidityProviderRewards": "Liquidity provider rewards",
   "liquidityProviderRewardsDesc": "Liquidity providers earn a 0.25% fee on all trades proportional to their share of the pool. Fees are added to the pool, accrue in real time and can be claimed by withdrawing your liquidity.",
   "liquidityProviderRewardsReadMore": "Read more about providing liquidity",
+  "legacyLiquidityProviderCardTitle": "Looking for your liquidity?",
+  "legacyLiquidityProviderCardBody": "We're seeing that you provided liquidity with our legacy interface. We recently upgraded our interface to let users deposit straight into a rewards contract when they provide liquidity, so you may not see your liquidity here.",
```
Comment (jophish) [inline public/locales/en.json]: nit:
```suggestion
  "legacyLiquidityProviderCardBody": "We're seeing that you provided liquidity with our legacy interface. We recently upgraded our interface to let users deposit straight into a rewards contract to earn additional auto-compouding rewards when they provide liquidity, so you may not see your liquidity here.",
```

File: public/locales/en.json [hunk 4]
```diff
@@ -98,6 +98,9 @@
...
+  "legacyLiquidityProviderCardCTA": "To recover your liquidity click here.",
```
Comment (jophish) [inline public/locales/en.json]: hyper-nit:
```suggestion
  "legacyLiquidityProviderCardCTA": "Click here to recover your liquidity.",
```

File: src/pages/ProvideLiquidity/index.tsx (hunk 1 above)
Comment (cajubelt) [reply src/pages/ProvideLiquidity/index.tsx]: it is because the change is not live on the deployed instance

Comment (cajubelt) [reply src/pages/ProvideLiquidity/index.tsx]: would be nicer to get the base URL dynamically, but I was hacking

File: src/pages/ProvideLiquidity/index.tsx (hunk 2 above)
Comment (cajubelt) [reply src/pages/ProvideLiquidity/index.tsx:89]: fine with this if u wanna do it but I am trying to wrap up for the night tbh

File: src/pages/ProvideLiquidity/index.tsx (hunk 1 above)
Comment (jophish) [reply src/pages/ProvideLiquidity/index.tsx]: :facepalm: of course lol

**Pull request merged**